Tools/build-swiftly-release/BuildSwiftlyRelease.swift

```diff
@@ -371,8 +354,8 @@ struct BuildSwiftlyRelease: AsyncParsableCommand {
 
         FileManager.default.changeCurrentDirectoryPath(cwd)
 
-        // Statically link standard libraries for maximum portability of the swiftly binary
-        try runProgram(swift, "build", "--product=swiftly", "--pkg-config-path=\(pkgConfigPath)/lib/pkgconfig", "--static-swift-stdlib", "--configuration=release")
+        try runProgram(swift, "build", "--swift-sdk", "\(arch)-swift-linux-musl", "--product=swiftly", "--pkg-config-path=\(pkgConfigPath)/lib/pkgconfig", "--static-swift-stdlib", "--configuration=release")
+        try runProgram(swift, "sdk", "remove", sdkName)
```

Comment:
Could this inadvertently remove the linux SDK from a users machine even if they didn't want it to? As in they already had the static sdk installed, built swiftly, and now their SDK is gone and maybe they didn't expect that?